### DISCUSSION TITLE
Killed show in console

### DISCUSSION THREAD

**Discussion by raisePiginYourhome** [mdolab/dafoam#490] [General]
Here are some information about the process being Killed.

Creating MRF zone list from MRFProperties
    creating MRF zone: MRF
Printing Primal Residual Statistics.
U Residual Norm2: (27668.6 24890.3 34576.1)
U Residual Mean: (23.7095 21.3805 30.0041)
U Residual Max: (2289.26 2141.75 1831.62)
p Residual Norm2: 2813.42
p Residual Mean: 1.48852
p Residual Max: 398.092
omega Residual Norm2: 5.65203e+07
omega Residual Mean: 31584.9
omega Residual Max: 8.62507e+06
k Residual Norm2: 4646.44
k Residual Mean: 3.66716
k Residual Max: 372.881
phi Residual Norm2: 3.72855
phi Residual Mean: 0.00174456
phi Residual Max: 0.425022
End

Nonlinear constraints
{'cruise.aero_post.functionals.MFR': array([-0.1909])}

Objectives
{'cruise.aero_post.functionals.CMZ': array([67.36871172211309])}

Computing partials for  ['CMZ']
Calculating dFdW using reverse-mode AD
AMI: Creating addressing and weights between 2139 source faces and 2139 target faces
AMI: Patch source sum(weights) min:0.999978 max:1 average:1
AMI: Patch target sum(weights) min:0.99998 max:1 average:1
    Adding all faces of patches 1(blade) ...
    Found matching patch blade with 6348 faces.
Calculating dFdXv using reverse-mode AD
AMI: Creating addressing and weights between 2139 source faces and 2139 target faces
AMI: Patch source sum(weights) min:0.999978 max:1 average:1
AMI: Patch target sum(weights) min:0.99998 max:1 average:1
    Adding all faces of patches 1(blade) ...
    Found matching patch blade with 6348 faces.
AMI: Creating addressing and weights between 2139 source faces and 2139 target faces
AMI: Patch source sum(weights) min:0.999978 max:1 average:1
AMI: Patch target sum(weights) min:0.99998 max:1 average:1
Killed

Then the process abort, what caused this situation to occur?
I think the question is there not has enough memory, is it? 
I used 32GB for four hundred thousand grids mesh, but I still get Killed in consloe, how to deal with it?

**Comment by friedenhe** [comment]
that is right, 32 GB is not enough for 400K cells. check the faq for recommended memory.
